scripts/three-sixty.js

```diff
@@ -311,28 +341,8 @@ H5P.ThreeSixty = (function (EventDispatcher, THREE) {
      * @param {Element} element
      * @return {THREE.CSS3DObject}
      */
-    self.find = function (element) {
-      for (let i = 0; i < threeElements.length; i++) {
-        if (threeElements[i].element === element) {
-          return threeElements[i];
-        }
-      }
-    };
-
-    /**
-     * Find the index of the given element.
-     * TODO: Move into a separate collection handling class
-     *
-     * @param {Element} element
-     * @return {number}
-     */
-    self.indexOf = function (element) {
-      for (let i = 0; i < threeElements.length; i++) {
-        if (threeElements[i].element === element) {
-          return i;
-        }
-      }
-    };
+    this.find = (element) => 
+      threeElements.find(threeElement => threeElement.element === element);
```

Comment:
There is already an established standard in the file using `self` for public functions. We do not allow developers to start new standards within the same file without refactoring everything for consistency.